LICENSE.txt

```diff
@@ -6,34 +6,39 @@ Most of the Checker Framework is licensed under the GNU General Public
 License, version 2 (GPL2), with the classpath exception.  The text of this
 license appears below.  This is the same license used for OpenJDK.
 
-A few parts of the Checker Framework have more permissive licenses.
+A few parts of the Checker Framework have more permissive licenses, notably
+the parts that you might want to include with your own program.
 
  * The annotations and utility files are licensed under the MIT License.
-   (The text of this license also appears below.)  More specifically, all
-   the parts of the Checker Framework that you might want to include with
-   your own program use the MIT License.  This is the checker-qual*.jar and
-   checker-compat-qual*.jar files and all the files that appear in them:
-   every file in a qual/ directory, plus utility files such as
-   NullnessUtil.java, RegexUtil.java, SignednessUtil.java, etc.  In
-   addition, the cleanroom implementations of third-party annotations,
-   which the Checker Framework recognizes as aliases for its own
-   annotations, are licensed under the MIT License.
-
-Some external libraries that are included with the Checker Framework have
-different licenses.
+   (The text of this license also appears below.)  This applies to the
+   checker-qual*.jar and all the files that appear in it:  every file in a
+   qual/ directory, plus utility files including FormatUtil.java,
+   I18nFormatUtil.java, NullnessUtil.java, Opt.java,
+   PurityUnqualified.java, RegexUtil.java, SignednessUtil.java, and
+   UnitsTools.java.  It also applies to the cleanroom implementations of
+   third-party annotations (e.g., in checker/src/testannotations/ and in
```

Comment:
I recommend not using "e.g." because, as above, it implies that additional files fall into this category. Again, just dropping that will make this sentence perfect. (Assuming that those are the only two instances.)